Library/Formula/procyon-decompiler.rb

```diff
@@ -0,0 +1,17 @@
+class ProcyonDecompiler < Formula
+  homepage "https://bitbucket.org/mstrobel/procyon/wiki/Java%20Decompiler"
+  url "https://bitbucket.org/mstrobel/procyon/downloads/procyon-decompiler-0.5.28.jar"
+  sha256 "356916d27f06c6c4fedfb48081a1b0a261aac4a6ec7f603ba3b322fc51f7d1e3"
+
+  depends_on :java => "1.7+"
+
+  def install
+    libexec.install "procyon-decompiler-#{version}.jar"
+    bin.write_jar_script libexec/"procyon-decompiler-#{version}.jar", "procyon-decompiler", "$@"
+  end
+
+  test do
+    system "#{bin}/procyon-decompiler", "-?"
```

Comment:
Good point. Actually found a bug while doing this :) I'll update the pull request when it's fixed.